docs/docs/concepts/structured_output.mdx

```diff
@@ -0,0 +1,120 @@
+# Structured output
+
+## Motivation 
+
+While many AI applications, such as chatbots, typically respond in natural language, there are scenarios where we need models to output in a structured format. 
+
+This is what we call structured output. Structured output is particularly useful when:
```

Comment:
nit: What does the "This is" referring to? (I think this sentence is basically saying "This structure format is what we call structured output"